File: python/python_oop_practice/p_oop_2/chap05/excercise/order.py
class Order:
    # 생성자를 통한 멤버변수 선언 및 초기화
    def __init__(self):
        self.order_id = None
        self.customer_id = None
        self.order_date = None
        self.customer_name = None
        self.product_id = None
        self.shipping_address = None


def main():
    order = Order()

    order.order_id = 201803120001
    order.customer_id = "abc123";
    order.order_date = "2018년 3월 12일";
    order.customer_name = "홍길순";
    order.product_id = "PD345-12";
    order.shipping_address = "서울시 영등포구 여의도동 20번지";

    print("주문 번호:", order.order_id)
    print("주문자 아이디:", order.customer_id)
    print("주문 날짜:", order.order_date)
    print("주문자 이름:", order.customer_name)
    print("주문 상품 번호:", order.product_id)
    print("배송 주소:", order.shipping_address)


if __name__ == '__main__':
    main()

File: java/doit_java/chapter5/src/alone/Order.java
/*
쇼핑몰에 주문이 들어왔습니다. 주문 내용은 다음과 같습니다.

주문 번호: 201803120001
주문자 아이디: abc123
주문 날짜: 2018년 3월 12일
주문자 이름: 홍길순
주문 상품번호: PD0345-12
배송 주소: 서울시 영등포구 여의도동 20번지

위 주문 내용에 대한 클래스를 만들과 주문 내요을 인스턴스로 생성한 후
위와 같은 형식으로 주문 내용을 그대로 출력해 보세요.
*/

package alone;

public class Order {

    long orderID;
    String customerID;
    String orderDate;
    String customerName;
    String productID;
    String shippingAddress;
}